extensions/stock-api-flask/README.md

```diff
@@ -0,0 +1,16 @@
+# Flask Stock Pricing Service
+
+## About this example
+
+An API allows you to turn your models into production services that other tools and teams can use. APIs are a great way for software engineering teams to use your models without translating them into different languages.
```

Comment:
These (preexisting) sentences in the README are… basically saying different flavors of the same thing.

```suggestion
APIs allows you to turn your models into production services that other tools and teams can use without translating them into different languages.
```